### PR TITLE
Counters for total tunnel usage and net result

### PR DESCRIPTION
**Pull request by Captain-Coder**
Snorberhuis never got to implementing front page counters for the tunnel community. I did them as a first assignment in Tribler. I'd kinda forgotten about them. I tidied up the status bar logic a little bit while I was at it. And a few nitpicks.

### CODE REVIEW

**Comment by synctext** [comment]
OK to test

**Comment by whirm** [comment]
retest this please

**Comment by synctext** [comment]
The counters "Total: " plus "Balance:" are good.

Are there tooltips when hovering over it?
like "Total bandwidth used for yourself and relaying encrypted traffic for privacy of others"

**Comment by devos50** [comment]
Nice addition, very nice to know how much traffic in total I proxied for others.

@synctext there are tooltips when you hover over the labels: "Total Anonymity Contribution" and "Anonymity Contribution Balance".

**Comment by whirm** [comment]
@devos50 does it look OK on you laptop too? (no time to test it myself)

**Comment by devos50** [comment]
@whirm https://gyazo.com/0b7320f8c0e799c78cd82bc31ad78d1b

**Comment by whirm** [comment]
Maybe it should be a bit more explicit? Total and balance what? Maybe some icon?
@synctext what do you say?

**Comment by whirm** [comment]
@synctext this is waiting for your feedback

**Comment by whirm** [comment]
retest this please

**Comment by synctext** [comment]
retest this please

**Comment by synctext** [comment]
add to whitelist

**Comment by synctext** [comment]
Looks good and ready to merge if passing.

**Comment by whirm** [comment]
retest this please

File: Tribler/community/bartercast4/statistics.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def close(self):
  UNIQUE (type, peer));
 
 CREATE TABLE option(key TEXT PRIMARY KEY, value BLOB);
-INSERT INTO option(key, value) VALUES('database_version', '""" + str(LATEST_VERSION) + """');
+INSERT INTO option(key, value) VALUES('database_version', '""" + str(LATEST_VERSION) + u"""');
```
Comment (whirm) [inline Tribler/community/bartercast4/statistics.py]: This doesn't look related to the pull request?

Comment (Captain-Coder) [inline Tribler/community/bartercast4/statistics.py]: Not really, i just noticed it and included it. Seems a bit overkill to put it in a separate pull request. Can do if you want it ofc...

**Comment by Captain-Coder** [comment]
retest this please

**Comment by whirm** [comment]
retest this please

File: Tribler/community/bartercast4/statistics.py (hunk 1 above)
Comment (whirm) [inline Tribler/community/bartercast4/statistics.py]: At least put it into its own commit, explaining the change.

Comment (whirm) [inline Tribler/community/bartercast4/statistics.py]: Also, if you find bugs, send them to `next` instead (even if it's a single line PR) otherwise they will not get into the next release.

**Comment by synctext** [comment]
passes, ready to merge.

**Comment by whirm** [comment]
I'm still waiting for the bugfix to be separated and sent to a PR in `next`.

**Comment by synctext** [comment]
@Captain-Coder please send the 1 char bug-fix to our stable branch...

**Comment by whirm** [comment]
@Captain-Coder should I wait for that bugfix to hit `next` before merging?  What is it it fixing?

**Comment by Captain-Coder** [comment]
Soory for the delay guys, I was rather sick. The 1 character doesn't fix anything, its just that the one string is unicode and it gets concatenated to a string that is not unicode. This is not uniform, I thought i'd just smooth that out. So it's not a fix, it's an improvement :)

**Comment by whirm** [comment]
Ready to merge then?

**Comment by Captain-Coder** [comment]
Yes
